package.json

```diff
@@ -33,17 +33,18 @@
   "scripts": {
     "dev": "nodemon",
     "start": "yarn build && cd build && node server.js",
-    "build": "rimraf ./build && tsc && rsync -a --exclude \"*/node_modules\" ./src/plugins ./build",
+    "build": "./utils/build_all.sh",
     "test": "jest --runInBand",
     "pg": "exec ./utils/postgraphile_launch.sh",
     "pg_permissions": "exec ./utils/postgraphile_launch_permissions.sh",
     "dev_pg": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg\"",
     "dev_pg_permissions": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg_permissions\"",
-    "database_init": "./database/initialise_database.sh && ./database/insert_data.sh && ./database/post_data_insert.sh",
-    "build_plugins": "exec ./utils/build_plugins.sh",
+    "database_init": "exec ./database/initialise_database.sh",
```

Comment:
Was this an intentional re-version of the database_init script? Don't you need that for the snapshots?

Reply:
That's my clumsy conflict resolution during merge from master:
48190f2a4a08ffcf254b7b08afec66c6f4297c4a, thanks for noticing